UKHomeOffice/cerberus-service
# COP-8927: Update tests to adapt latest changes

## Description
Automation tests updated to adapt the latest changes.

## To Test
npm run cypress:test:report -- -b electron

check the generated report on folder,  `mochawesome-report/mochawesome.html`


## Developer Checklist

\* Required
